### ISSUE TITLE
PropertySortOrder not honoured

### ISSUE THREAD

**Issue by andykenward** [adamwalzer/vscode-scss-lint#1]
Doesn't honour the PropertySortOrder rule.

example rule
```
  PropertySortOrder:
    enabled: true
    order:
      - content
      - z-index
      - float
      - clear
```

**Comment by adamwalzer** [comment]
Please try

```
    PropertySortOrder:
        severity: error
        order:
            - content
            - z-index
            - float
            - clear
```

and let me know if that works. Thanks!

**Comment by adamwalzer** [comment]
Also be aware this will only throw an error if you have one of those properties in your selector. If you have something like
```
.great-class-name {
  left: 0;
  top: 0;
}
```

You will not get an error, but if you have
```
.great-class-name {
  left: 0;
  top: 0;
  z-index: 1;
}
```
you should get an error.

**Comment by adamwalzer** [comment]
The real issue here was that the extension did not report warnings. It only supported errors. I have fixed that so this issue should be resolved in version 0.0.11.

**Comment by chialin** [comment]
It works in version 0.0.11. thank you.

**Comment by adamwalzer** [comment]
Awesome. I'm going to close this issue. Thanks!
